spec/player_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'play'
+
+describe Player do
+  subject(:naomi) { Player.new('Naomi') }
```

Comment:
Sorry, why did you do this test as well?


Asking because I don't know if we should have, rather than telling you you did anything wrong ^-^